src/openrct2/interface/Console.cpp

```diff
@@ -231,21 +239,32 @@ void console_draw(rct_drawpixelinfo *dpi)
         memcpy(lineCh, ch, lineLength);
         lineCh[lineLength] = 0;
 
-        gfx_draw_string(dpi, lineBuffer, COLOUR_LIGHT_PURPLE | COLOUR_FLAG_OUTLINE | COLOUR_FLAG_INSET, x, y);
```

Comment:
Do you recall why this was using 'light purple' to begin with? I guess it's 'bright' enough to be perceived as white, but this is a little confusing, haha.

Reply:
I have no idea, I naively assumed there was a reason! Will try changing it.

Reply:
No difference really - will change to `COLOUR_WHITE` by default, and will probably make it themeable.